packages/@react-spectrum/form/docs/Form.mdx

```diff
@@ -0,0 +1,224 @@
+  <TextField label="Password" placeholder="1234" />
+  <Checkbox>Remember me</Checkbox>
+</Form>
+```
+
+## Content
+
+Form accepts one or more children to render as its form elements. Label alignment, positioning, and other [properties](#visual-options)
+are set on the Form itself and propagated to its children to generate a cohesive design. To override these top level Form props on individual children, 
+set them directly on the child instead.
+
+```tsx example
+<h3 id="label-2">Sign Up</h3>
+<Form width={400} aria-labelledby="label-2" isRequired necessityIndicator="label">
+  <TextField label="Name" placeholder="Bob Doe" />
+  <TextField label="Email" placeholder="abc@adobe.com" />
+  <TextField label="Address" placeholder="123 Any Street" isRequired={false} />
+</Form>
+```
+
+### Accessibility
+
+As per [W3 Form guidelines](https://www.w3.org/TR/wai-aria-practices/examples/landmarks/form.html), the controls provided to the form should 
+use native host semantics (`button`, `input`, `select`, `textarea`) whenever possible.
+
+## Labeling
+
+### Accessibility
+
+If a label is provided to the Form by adding either the `aria-label` or `aria-labelledby` prop, 
+the element will be identified to assistive technology as a form landmark region. 
+
+```tsx example
+<h3 id="label-3">Personal Information</h3>
+<Form width={400} aria-labelledby="label-3">
+  <TextField label="First Name" placeholder="Bob" />
+  <TextField label="Last Name" placeholder="Dole" />
+  <RadioGroup label="Favorite pet" name="favorite-pet-group">
+    <Radio value="dogs">Dogs</Radio>
+    <Radio value="cats">Cats</Radio>
+    <Radio value="dragons">Dragons</Radio>
+  </RadioGroup>
+</Form>
+```
+
+
+## Props
+
+<PropTable component={docs.exports.Form} links={docs.links} />
+
+## Visual Options
+
+### Label Position
+
+```tsx example
+<Form width={400} marginBottom="20px" labelPosition="top">
+  <TextField label="Name" placeholder="John Smith" />
+  <RadioGroup label="Favorite pet" name="favorite-pet-group">
+    <Radio value="dogs">Dogs</Radio>
+    <Radio value="cats">Cats</Radio>
+  </RadioGroup>
+</Form>
+
+<Form width={400} labelPosition="side">
+  <TextField label="Name" placeholder="John Smith" />
+  <RadioGroup label="Favorite pet" name="favorite-pet-group">
+    <Radio value="dogs">Dogs</Radio>
+    <Radio value="cats">Cats</Radio>
+  </RadioGroup>
+</Form>
+```
+
+### Label Alignment
+
+```tsx example
+<Form width={400} marginBottom="20px" labelAlign="start">
+  <TextField label="Name" placeholder="John Smith" />
+  <RadioGroup label="Favorite pet" name="favorite-pet-group">
+    <Radio value="dogs">Dogs</Radio>
+    <Radio value="cats">Cats</Radio>
+  </RadioGroup>
+</Form>
+
+<Form width={400} labelAlign="end">
+  <TextField label="Name" placeholder="John Smith" />
+  <RadioGroup label="Favorite pet" name="favorite-pet-group">
+    <Radio value="dogs">Dogs</Radio>
+    <Radio value="cats">Cats</Radio>
+  </RadioGroup>
+</Form>
+```
+
+### Quiet
```

Comment:
I feel like the quiet and emphasized examples aren't needed because they aren't interesting examples like label position or label align.

Reply:
_shrug_ I'm fine with removing it or keeping it, I'll let the majority decide. Just wanna be consistent

Reply:
IMO these are important to keep.